pajbot/bot.py

```diff
@@ -269,6 +269,8 @@ def __init__(self, config, args):
             "molly_age_in_years": self.c_molly_age_in_years,
         }
 
+        self.user_agent = f"pajbot/{VERSION} ({self.nickname})"
```

Comment:
user-agent should be consistent with the old urlfetch format: `f"pajbot1/{VERSION} ({bot.nickname})"`